docs/gaia/gaia.rst

```diff
@@ -807,81 +865,73 @@ Then, any user belonging to that group will be able to user your shared table in
 
-2.7. Epoch photometry access (datalink)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Epoch photometry data are retrieved as a table from the Gaia archive.
+Once you have the required identifiers ('ids'), you can retrieve the actual data by typing:
 
-In order to download epoch photometry data, you need to know the identifiers of the sources you are interested in.
+.. code-block:: python
 
-So, the first step, is to execute a query to obtain the identifiers, and then you can retrieve the data.
+  >>> # Retrieve data products
+  >>> all = Gaia.load_data(ids=ids, format='fits', data_release='Gaia DR2')
 
-.. code-block:: python
+  Retrieving data.
+  Done.
 
-  >>> from astroquery.gaia import Gaia
-  >>> query = "SELECT TOP 500 * FROM gaiadr2.gaia_source  \
-  >>> WHERE CONTAINS(POINT('ICRS',gaiadr2.gaia_source.ra,gaiadr2.gaia_source.dec),\
-  >>> CIRCLE('ICRS',COORD1(\
-  >>> EPOCH_PROP_POS(339.8049024487712,64.8585025696523,2.3585,92.7710,190.7920,.3000,2000,2015.5)),\
-  >>> COORD2(EPOCH_PROP_POS(339.8049024487712,64.8585025696523,2.3585,92.7710,190.7920,.3000,2000,2015.5)),0.001388888888888889))=1" 
-  >>>
-  >>> job = Gaia.launch_job(query)
-  >>>
-  >>> results = job.get_results()
-  >>> ids=results['source_id']
-  >>> print(ids)
-  >>>
-  >>> # Retrieve epoch photoemtry data
-  >>> epoch_photometry_data = Gaia.load_data(ids=ids, retrieval_type="epoch_photometry")
-  >>> print("source id \t\tBand \t\tMag \t\tTime \t\tFlux \t\tFluxError")
-  >>> for source, band, mag, time, flux, flux_error in zip(epoch_photometry_data['source_id'], \
-  >>>                          epoch_photometry_data['band'], \
-  >>>                          epoch_photometry_data['mag'],\
-  >>>                          epoch_photometry_data['time'],\
-  >>>                          epoch_photometry_data['flux'],\
-  >>>                          epoch_photometry_data['flux_error']):
-  >>>    print(str(source) + "\t"+  str(band, 'utf-8') + "\t\t" + str(mag) + "\t" + str(time) + "\t" + str(flux) + "\t" + str(flux_error))
+  List of products available:
+  Product = RVS_SPECTRA-Gaia DR2 4203426768245304192.fits
+  Product = RVS_SPECTRA-Gaia DR2 4257750098718805248.fits
+  Product = XP_BASIS-Gaia DR2 4203426768245304192.fits
+  Product = MCMC-Gaia DR2 4257750098718805248.fits
+  Product = XP_SPECTRA-Gaia DR2 4257750098718805248.fits
+  Product = EPOCH_PHOTOMETRY-Gaia DR2 4257750098718805248.fits
+  Product = XP_BASIS-Gaia DR2 4257750098718805248.fits
+  Product = MCMC-Gaia DR2 4203426768245304192.fits
+  Product = XP_SPECTRA-Gaia DR2 4203426768245304192.fits
+
+  >>> # Get one data product above listed
+  >>> all['RVS_SPECTRA-Gaia DR2 4203426768245304192.fits']
+
+  <Table length=2400>
+
+  wavelength    flux     flux_error
+  nm
+  float64    float32     float32
+  ---------- ---------- -----------
+  846.0   1.1543361  0.14655493
+  846.01  1.0716769  0.09160478
+
+  ...        ...         ...
+
+  869.77  1.0126361  0.03356205
+  869.78  1.0286258 0.035302058
```

Comment:
This example doesn't work for me. `all = Gaia.load_data(ids=ids, format='fits', data_release='Gaia DR2')` does not print any information for me, when I add `verbose=True` I get:
```
Retrieving data.
Data request: VALID_DATA=false&ID=2211629018927324288&RELEASE=Gaia+DR2&DATA_STRUCTURE=INDIVIDUAL&FORMAT=fits&RETRIEVAL_TYPE=ALL&USE_ZIP_ALWAYS=true
------>https
host = gea.esac.esa.int:443
context = /data-server/data
Content-type = application/x-www-form-urlencoded
200 200
Reading...
Done.
``` 

`all['RVS_SPECTRA-Gaia DR2 4203426768245304192.fits']` gives me a KeyError, and if I do `all.keys()` I see:
```
dict_keys(['EPOCH_PHOTOMETRY-Gaia DR2 2211629018927324288.fits'])
```
So then I can do `all['EPOCH_PHOTOMETRY-Gaia DR2 2211629018927324288.fits']` at which point I see:
```
[<Table length=90>
      source_id          transit_id    band ... other_flags    solution_id    
                                            ...                               
        int64              int64       str2 ...    int64          int64       
 ------------------- ----------------- ---- ... ----------- ------------------
 2211629018927324288 16577507605788211   G  ...        4097 369295549951641967
 2211629018927324288 16577507605788211   G  ...        4097 369295549951641967
 2211629018927324288 16577507605788211   G  ...        4097 369295549951641967
...
```